domains/certificates/RegistryConnector/Worker/Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
+ARG SDK_VERSION
+ARG RUNTIME_VERSION
+FROM mcr.microsoft.com/dotnet/aspnet:${RUNTIME_VERSION}-alpine3.17 AS base
 WORKDIR /app
 EXPOSE 80
 EXPOSE 443
 
-RUN apt-get -y update && apt-get -y install wget
-HEALTHCHECK CMD wget -qO- http://localhost/health > /dev/null || exit 1
-
-FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:${SDK_VERSION}-alpine3.17 AS build
```

Comment:
This cannot be an alpine image due to our usage of the PO registry client, which does not support MUSL.

Reply:
Fixed